### PR TITLE
Improve README to use virtual env

### PR DESCRIPTION
**Pull request by VictorRodriguez**
This patch fis the need of jinja2 instead of jinja
Also give a bit more of description on the README

Signed-off-by: VictorRodriguez <vm.rod25@gmail.com>

### CODE REVIEW

**Comment by Meadosc** [comment]
@unrahul @VictorRodriguez Is it appropriate to specify what virtual environment a user utilizes? This seems outside the scope of this repository.

**Comment by VictorRodriguez** [comment]
@Meadosc maybe it was useful for me to follow these instructions I can split the commit to just the fix for jinja, in CLR and MacOS is not working w/o the fix. Also the typo

File: README.md [hunk 1]
```diff
@@ -11,15 +11,62 @@ A prototype for a tool to generate Dockerfiles from templates and scripts for st
...
+
+Python “Virtual Environments” allow Python packages to be installed in an
+isolated location for a particular application, rather than being installed
+globally.
+
+### Installation of Virtual Environment
+Make sure you have python **virtualenv** package installed in your host machine.
+
+```
+ $ sudo apt install python-pip
```
Comment (unrahul) [inline README.md]: may be not add OS specific things, or specify for which OS.

**Comment by Meadosc** [comment]
It would be better to keep commits logically separated. With an exception for typos, in my opinion.

**Comment by unrahul** [comment]
Let;s merge this and take care of the OS specific install as a follow up.

**Pull request merged**